package.json

```diff
@@ -2,12 +2,15 @@
   "name": "search-transport",
   "version": "0.1.0",
   "description": "a tiny search engine",
+  "engines": {
+    "node": "^14"
+  },
   "main": "src/index.js",
   "files": [
     "src/index.js"
   ],
   "scripts": {
-    "build": "rm -r build; tsc",
+    "build": "rm -r build; tsc && npm run format",
```

Comment:
It's possible to have lint-rejected files in the build directory - formatting as part of the build steps ensures that none get accidentally commited.

Necessary now, not strictly necessary if we move to build artifacts as part of the CD process.